Branch-SDK-Automation-TestBed/src/main/java/io/branch/branchandroiddemo/Constants.java

```diff
@@ -33,4 +19,7 @@ public interface Constants {
     String AFTER_READ_DEEP_LINK = "AfterReadDeepLink";
     String TRACK_CONTENT_DATA = "TrackContentData";
     String CREATE_QR_CODE = "CreateQRCode";
+    String SET_DMA_Params = "SetDMAParams";
+    String INIT_SESSION = "InitSession";
+    String UNKNOWN = "Unknown";
```

Comment:
When would this be used?

Reply:
https://github.com/BranchMetrics/android-branch-deep-linking-attribution/blob/f25a68fd21f61f9b97cb9ec25b56ea90308a8b8d/Branch-SDK-Automation-TestBed/src/main/java/io/branch/branchandroiddemo/activities/LogDataActivity.java#L37